src/components/button/button.tsx

```diff
@@ -67,12 +67,14 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
         {loading ? <Spinner size="small" /> : null}
 
         {LeftIcon && !loading ? (
-          <LeftIcon className={iconVariants[type]} />
+          <LeftIcon className={`${iconVariants[type]} w-[1em] h-[1em]`} />
         ) : null}
 
         {children}
 
-        {RightIcon ? <RightIcon className={iconVariants[type]} /> : null}
+        {RightIcon ? (
+          <RightIcon className={`${iconVariants[type]} w-[1em] h-[1em]`} />
```

Comment:
In our design file the icon for button has a 12px bounding box. So you can use h-3 and w-3 and we are good.